static/js/common.ts

```diff
@@ -56,26 +56,40 @@ export function adjust_mac_shortcuts(key_elem_class: string, require_cmd_style =
     const keys_map = new Map([
         ["Backspace", "Delete"],
         ["Enter", "Return"],
-        ["Home", "Fn + ←"],
-        ["End", "Fn + →"],
-        ["PgUp", "Fn + ↑"],
-        ["PgDn", "Fn + ↓"],
+        ["Home", "←"],
+        ["End", "→"],
+        ["PgUp", "↑"],
+        ["PgDn", "↓"],
         ["Ctrl", "⌘"],
     ]);
 
+    const fn_shortcuts = new Set(["Home", "End", "PgUp", "PgDn"]);
+
+    let key_elem = "code";
+
+    if (key_elem_class.slice(Math.max(0, key_elem_class.lastIndexOf(" ") + 1)) === "kbd") {
+        key_elem = "kbd";
+    }
+
     $(key_elem_class).each(function () {
         let key_text = $(this).text();
-        const keys = key_text.match(/[^\s+]+/g) || [];
 
-        if (key_text.includes("Ctrl") && require_cmd_style) {
+        if (/\s/.test(key_text)) {
+            return;
+        }
+
+        if (key_text === "Ctrl" && require_cmd_style) {
             $(this).addClass("mac-cmd-key");
         }
 
-        for (const key of keys) {
-            const replace_key = keys_map.get(key);
-            if (replace_key !== undefined) {
-                key_text = key_text.replace(key, replace_key);
-            }
+        if (fn_shortcuts.has(key_text)) {
+            const fn_key = "<" + key_elem + ">Fn</" + key_elem + "> + ";
```

Comment:
This is probably not the right bay to build HTML here in JS; I think it might actually be cleaner to just have an `if/else` for the new parameter suggested above, and just hardcode `<kbd>Fn</kbd>` or `<code>Fn</code>` depending on that parameter.

Reply:
Updated and extended the node testing to cover both potential values of the new parameter.